src/sas/sascalc/pr/p_invertor.py

```diff
@@ -353,13 +349,10 @@ def get_iq_smeared(self, pars, q):
         q = np.atleast_1d(q)
         pars = np.float64(pars)
 
-
         npts = 21
```

Comment:
I know this hasn't been touched but magic numbers are always bit scary... Do we know why is it 21?

Reply:
Number of points in the resolution integral.  Presumably chosen to balance speed and accuracy over the expected range of inputs.  I don't know how this particular number was chosen.

Reply:
Should that be a comment added to the code for future developers?